### DISCUSSION TITLE
Is there an official Radicale docker image?

### DISCUSSION THREAD

**Discussion by instantdreams** [Kozea/Radicale#1381] [Q&A general]
There are a number of [Radicale docker images](https://hub.docker.com/search?q=radicale) available on the docker hub. The [downloads section](https://radicale.org/v3.html#linux-distribution-packages) has a reference to a Dockerfile.

Is there an official Radicale docker image available? Or perhaps a trusted image?

**Comment by gregorko** [comment]
My favorite docker image: tomsquest/docker-radicale
Using it since many years.
